### ISSUE TITLE
When merging to an existing rule, no properties should be overwritten

### ISSUE THREAD

**Issue by gerrycampion** [cdisc-org/crbuilder#2]
Example
UID: 0066ab1c-982c-42e6-96f8-e7cbec162a91

In prod, the outcome is:
```yaml
Outcome:
  Message: USUBJID is not unique within study
  Output Variables:
    - USUBJID
```

In dev, the outcome is overwritten:
```yaml
Outcome:
  Message: ' and USUBJID unique within a submission'
```

**Comment by htu** [comment]
Do you have Rule ID of this document? I have fixed the issue and would like to run a test.

**Comment by gerrycampion** [comment]
@htu, I included the id in the ticket. You can use this to find the rule or rule id in the rule editor. The last column has a uid filter.

**Comment by htu** [comment]
Got it and it is CG0151

**Comment by gerrycampion** [comment]
The outcome message is now overwritten by an empty string, which we don't want.

**Comment by htu** [comment]
This has been fixed in v0.4.1
